sectorbuilder_test.go

```diff
@@ -195,9 +204,6 @@ func TestSealAndVerify(t *testing.T) {
 }
 
 func TestSealPoStNoCommit(t *testing.T) {
-	if testing.Short() {
```

Comment:
Removing this and the one below seems unrelated to this PR. Was this intentional?

Reply:
> Was this intentional?

There aren't any tests in this Git repo which qualify as "short," so IMHO there's no utility in that check.